README.md

```diff
@@ -150,6 +150,24 @@ The items in `hosts` list variable can contain the following parameters:
 | hosted_engine | UNDEF            | Specifies if the host is 'deploy' as hosted engine. |
 | power_management | UNDEF            | The power managment. You can choose a predefined variables, see the tables below. |
 
+In case you cannot use `hosts` variable for whatever reason in your playbook, you can change this variable's name
+by overriding value of `hosts_var_name` variable. Example:
+```yaml
+- name: Set up oVirt infrastructure
+  hosts: engine
+
+  roles:
+    - role: ovirt.infra
+      vars:
+        hosts_var_name: rhv_hosts
```

Comment:
Please don't use rhv in upstream code, let's rename to `ov_hosts` or `ovirt_hosts`